src/components/TodayText/TodayText.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {Text} from 'react-native';
+
+export default class TodayText extends React.Component {
+
+    render() {
+        const font = {fontFamily: 'WaitingForTheSunrise'};
+        const propStyles = this.props.style || {};
+        style = Object.assign(propStyles, font);
+
```

Comment:
set the font, get the style passed in from props if there is one, jam them into one object we can use for style.